internal/scorecard/alpha/tests/olm.go

```diff
@@ -59,6 +63,46 @@ func CRDsHaveResourcesTest(bundle registry.Bundle) scapiv1alpha2.ScorecardTestRe
 	r.Errors = make([]string, 0)
 	r.Suggestions = make([]string, 0)
 
+	csv, err := bundle.ClusterServiceVersion()
+	if err != nil {
+		r.Errors = append(r.Errors, err.Error())
+		r.State = scapiv1alpha2.ErrorState
+		return r
+	}
+
+	r.Log += fmt.Sprintf("Loaded ClusterServiceVersion: %s\n", csv.GetName())
+
+	apiCSV, err := registryToAPICSV(csv)
+	if err != nil {
+		r.Errors = append(r.Errors, err.Error())
+		r.State = scapiv1alpha2.ErrorState
+		return r
+	}
+	return CheckResources(apiCSV.Spec.CustomResourceDefinitions, r)
+}
+
+func registryToAPICSV(csv *registry.ClusterServiceVersion) (*operators.ClusterServiceVersion, error) {
```

Comment:
Will remove this function once https://github.com/operator-framework/operator-sdk/pull/2929 gets merged.